maestro-cli/src/main/java/maestro/cli/device/PickDeviceInteractor.kt

```diff
@@ -47,13 +54,22 @@ object PickDeviceInteractor {
     }
 
     private fun startDevice(): Device {
-        val availableDevices = DeviceService.listAvailableForLaunchDevices()
+        PrintUtils.message("No devices found. Would you like to start a new one? y/n")
+        val proceed = readlnOrNull()?.lowercase()?.trim()
+        if (proceed == "no" || proceed == "n") {
+            throw CliError("Please either start a device manually or via Maestro to proceed running your flows")
```

Comment:
Does starting via Maestro refer to running `maestro start-device`? If so, let's update the error message to be explicit here

```suggestion
            throw CliError("Please either start a device manually or via running maestro start-device to proceed running your flows")
```